### PR TITLE
Slides for first lesson

### PR DESCRIPTION
**Pull request by oktupol**
@ReDI-Munich-Frontend/frontend1 I thought of including a "Meet the teachers" slide which we show while introducing ourselves, and in it, I would include profile pictures. Would you be okay with it? If yes, can you send me one that I can include?

image

TODOs

- [x] Remove placeholder images

### CODE REVIEW

**Comment by oktupol** [comment]
- [ ] Removed placeholder images

**Review by guicheffer — APPROVED** [review]
This looks perfect for the first class. Looking forward to see this being taught.

File: src/lessons/01_getting_started/getting-started.slides.html [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+          <section data-auto-animate data-timing="30">
+            <h2>Git and Github</h2>
+            
+            <div class="git-github-comparison">
+              <div>
+                <div class="git" title="Git"><div>Git</div></div>
+                <div class="is-to">is to</div>
+                <div class="github" title="Github"><div>Github</div></div>
+                
+                <div class="what fragment" data-fragment-index="0">what</div>
```
Comment (guicheffer) [inline src/lessons/01_getting_started/getting-started.slides.html:217]: what do you think about changing into `whereas` or even `while` instead?

```suggestion
                <div class="what fragment" data-fragment-index="0">while</div>
```

Comment (oktupol) [reply src/lessons/01_getting_started/getting-started.slides.html:217]: I don't think that works logically 😅 I'm making a metaphor here. "while" or "whereas" indicates that there are differences though, doesn't it?

**Pull request merged**